examples/tinywl-new/OutputMenuBar.qml

```diff
@@ -102,6 +102,16 @@ Item {
                 }
             }
 
+            ToolButton {
+                text: "New Workspace"
+                onClicked: Helper.workspace.createContainer("Workspace"+Math.random());
```

Comment:
Why is random, should is a auto-increment number

Reply:
It's random in kwin
```
Id_1=9aaa9c14-b594-49b4-b52a-b2b72fbad632
Id_2=a82c8ff4-20ca-4d96-bce3-2e6489b0a62b
Id_3=f00821b4-f311-4a13-90a0-ee5c39e3daab
Id_4=b2531c33-bf31-4338-a676-a3b28351d9d2
```
Keep auto-increment number should be meaningless，Aready have an `index`